docs/EVG.md

```diff
@@ -180,6 +180,10 @@ Capitalized Terms are defined in the Baseline Requirements except where provided
 
 **Extended Validation Certificate:**  See EV Certificate.
 
+**Global Legal Entity Identifier Foundation:** Established by the Financial Stability Board in June 2014, the Global Legal Entity Identifier Foundation (GLEIF) is tasked to support the implementation and use of the Legal Entity Identifier (LEI). The foundation is backed and overseen by the LEI Regulatory Oversight Committee, representing public authorities from around the globe that have come together to jointly drive forward transparency within the global financial markets. GLEIF is a supra-national not-for-profit organization headquartered in Basel, Switzerland.
```

Comment:
None of this marketing copy is necessary. This provides no definitional value, especially for the purpose of the EVGs.

Suggestion: Delete Entirely.